share/spice/meetup/meetup.js

```diff
@@ -0,0 +1,45 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_meetup = function(api_result){
+
+        if (!api_result || api_result.error || !api_result.results) {
+            return Spice.failed('meetup');
+        }
+
+        Spice.add({
+          id: "meetup",
+          name: "meetup",
+          data: api_result.results,
+
+          meta: {
+              count: api_result.results.length,
+              itemType: 'meetups',
+              sourceName: "Meetup.com",
```

Comment:
We usually drop the last part of the `sourceName` (the technical name is escaping me right now but it's the ".com" part) like in https://duckduckgo.com/?q=ray+bradbury+books&ia=products we just say "Amazon" instead of "Amazon.com". So here, it would be just "Meetup. Ah, it's TLD. Just remembered.